index.html

```diff
@@ -1569,6 +1569,13 @@ <h2>Capabilities</h2>
   <td>Describes the <a>current session</a>’s <a>user prompt handler</a>.
    Defaults to the <a>dismiss and notify state</a>.
  </tr>
+
+ <tr>
+  <td><dfn>User Agent</dfn>
+  <td>"<code>userAgent</code>"
+  <td>string
+  <td>Provides the <a>remote end</a>'s <a>default User-Agent value</a>.
```

Comment:
To be in sync with the other application / os specific capabilities:

```suggestion
  <td>Identifies the <a>remote end</a>'s <a>default User-Agent value</a>.
```